docs/examples/voting.rst

```diff
@@ -194,4 +194,6 @@ Possible Improvements
 =====================
 
 Currently, many transactions are needed to assign the rights
-to vote to all participants. Can you think of a better way?
+to vote to all participants, and ``winningProposal()`` will
+not work correctly if two or more proposals are tied in voteCount.
```

Comment:
not sure I agree with the wording "will not work correctly". It's still returning "a" winning proposal

Reply:
Good point, will update it with something clearer